packages/oidc-dpop-client-browser/src/dpop/dpop.ts

```diff
@@ -30,14 +43,111 @@ import { JWK } from "node-jose";
  * @hidden
  */
 export async function generateJWK(
```

Comment:
Another niggle, but our naming convention means this should be named `generateJwk()`.

Reply:
Fixed in 86755fc7ab85dc1b8cbbe6c6ea5c0d1d4c57e3b6